### PR TITLE
Update Ingress documentation to link to examples for authenticated ingress

### PR DESCRIPTION
**Pull request by RoryStokes**
### What changes were proposed in this pull request?
A revision of the documentation on ingress to provide explicit information on how to secure an ingress with authentication.

### Why are the changes needed?
Existing documentation just had a note that it could be done. Linking to the official Kubernetes documentation for authenticated ingress seems valuable.

### Does this PR introduce any user-facing change?
No, purely documentation.

### How was this patch tested?
Purely documentation.

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by olofwalker** [comment]
@RoryStokes You have to agree to the CLA!

**Comment by RoryStokes** [comment]
> @RoryStokes You have to agree to the CLA!

Sorry about that, I'd initially pushed a commit with a different email address; it should be correct now

File: docs/shared-content-source/docs/modules/administration/pages/providing-external-access-to-cloudflow-services.adoc [hunk 1]
```diff
@@ -64,6 +62,17 @@ sensor-data-http-ingress   *       82.196.11.250       80        59s
 
 The ingress now has an address and data can be sent to the ingress on the address `82.196.11.250/sensor-data`
 
+== Authenticated ingress
+An _Ingress_ opens up a permanent, unauthenticated route by default. Kubernetes does not provide any authentication on ingresses. Authentication has to be implemented by the exposed service or by the ingress controller.
+
+To secure an Ingress for an otherwise open service to only allow access to authenticated users, the ingress resource (and in some cases the ingress controller itself) can be customised.
```
Comment (olofwalker) [inline docs/shared-content-source/docs/modules/administration/pages/providing-external-access-to-cloudflow-services.adoc:68]:
```suggestion
The following section in the Nginx ingress controller documentation shows how to use basic authentication or an `OAuth` sign-in:
```

File: docs/shared-content-source/docs/modules/administration/pages/providing-external-access-to-cloudflow-services.adoc [hunk 2]
```diff
@@ -64,6 +62,17 @@ sensor-data-http-ingress   *       82.196.11.250       80        59s
...
+
+For basic authentication or simple oauth sign-in, the ingress resource can be customised following the Kubernetes examples found at the following link.
```
Comment (olofwalker) [inline docs/shared-content-source/docs/modules/administration/pages/providing-external-access-to-cloudflow-services.adoc:70]:
```suggestion
```

**Comment by RayRoestenburg** [comment]
Thanks @RoryStokes !

**Pull request merged**